AY2021S2-CS2103-T14-1/tp
# Docs(UG): Fill up SunRez UI section

## What this does
This PR:
1. adds a picture showing the different sections of the GUI
2. adds some paragraphs describing how to interact with the SunRez UI via commands
3. renames section `About this Guide` to `SunRez User Interface (UI)`
4. capitalizes the `p` in the `Command parsing` section header

Closes #298 

## How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Check that the UG has the changes described above, and that they make sense.
